object/object.go

```diff
@@ -80,6 +82,15 @@ func Hashable(o Object) bool {
 	return false
 }
 
+func UnwrapHashable(o Object) any {
+	switch o.Type() { //nolint:exhaustive // getting a bit tired of this linter, why does it trigger when you have a default?
```

Comment:
I hate this linter, I always disable it

Reply:
I'd like to have it only when there is no "default" - also I'd like it to work on type switch but it doesn't

I should disable it or find how to tune it so default makes it shut up

Reply:
AH!     default-signifies-exhaustive: true
https://golangci-lint.run/usage/linters/#exhaustive

Reply:
https://github.com/fortio/workflows/pull/68